From a6469eb1d4529ec45e83e8307c0503c7c7ef1721 Mon Sep 17 00:00:00 2001
From: Frans-Willem Hardijzer <fw@hardijzer.nl>
Date: Sat, 6 Dec 2014 20:30:03 +0100
Subject: [PATCH] Stretch up maximum UART baudrate for AR933x.

Even though for some baudrates the timing error gets too high, it shouldn't prevent other baudrates that are perfectly fin (like 5mbaud) to be forbidden.
---
 drivers/tty/serial/ar933x_uart.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/tty/serial/ar933x_uart.c b/drivers/tty/serial/ar933x_uart.c
index 0be1c45..f3f819f 100644
--- a/drivers/tty/serial/ar933x_uart.c
+++ b/drivers/tty/serial/ar933x_uart.c
@@ -38,7 +38,7 @@
 #define AR933X_UART_MAX_STEP	0xffff
 
 #define AR933X_UART_MIN_BAUD	300
-#define AR933X_UART_MAX_BAUD	3000000
+#define AR933X_UART_MAX_BAUD	25000000
 
 #define AR933X_DUMMY_STATUS_RD	0x01
 
